code/mob/living/critter/voidhound.dm

```diff
@@ -0,0 +1,61 @@
+//Quick, stealthy, squishy assassin
+//Take the heat off of the wraith by attracting attention
+
+/mob/living/critter/voidhound
+	name = "Voidhound"
+	desc = "You probably shouldn't be staring at this thing."
+	density = 1
+	hand_count = 1
+	can_help = 1
+	can_throw = 1
+	can_grab = 0
+	can_disarm = 1
+	custom_gib_handler = /proc/gibs
+	icon = 'icons/mob/wraith_critters.dmi'
+	icon_state = "voidhound"
+	icon_state_dead = "dead_voidhound"
+	var/health_brute = 40
+	var/health_brute_vuln = 0.7
+	var/health_burn = 40
+	var/health_burn_vuln = 1
```

Comment:
what's with these being defined here

Reply:
I added those definitions to critter.dm as a parent.